### ISSUE TITLE
The outputs of madis2nc have more data

### ISSUE THREAD

**Issue by hsoh-u** [dtcenter/MET#1511]
The nightly build (2020-10-06) failed at madis2nc. The new outputs have more data.

DEBUG 3: Record Number: 5000
DEBUG 3:     [ACCEPT] seaLevelPress: value = 3.40282e+38, qc = Z

This data should be rejected because it's _FillValue for the seaLevelPress variable.

## Describe the Problem ##
The madsi2nc outputs have more data after #1345. The "get_nc_att" returned two different values: NcAtt object and the value of attribute. To avoid the confusion, "get_nc_att" which returns the value of the attribute was renamed to "get_nc_att_value" The calls of "get_nc_att" at madis2nc were not replaced with new one.

### Expected Behavior ###
Pass the madis2nc unittest or the filtering out the _FillValues with madis2nc.
For example, madis2nc_multiple_inputs, the last unittest for madsi2nc which the command is 
```
./madis2nc \
   /d3/projects/MET/MET_test_data/unit_test/obs_data/madis/metar/metar_2012040912_F000.nc \
  /d3/projects/MET/MET_test_data/unit_test/obs_data/madis/metar/metar_20170118_0000.nc   \
  metar_2012040912_20170118.nc   -type metar -rec_beg 5000 -v 2

should produce the output of the dimension

        nobs = 52548 ;
        nhdr = 7000 ;
 instead of 
        nobs = 105231 ;
        nhdr = 7007 ;
```

### Environment ###
Describe your runtime environment:
1. Linux Workstation
2. OS: RedHat Linux
3. Software version number(s): MET 10.0 develop

### To Reproduce ###
Describe the steps to reproduce the behavior:
1. Run the command above and check the dimensions

### Relevant Deadlines ###
NONE.

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [x] Select **scientist(s)** or **no scientist** required: None

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [x] Select **milestone** to relevant bugfix version

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Bugfix Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **main_\<Version>**.
Branch name: `bugfix_<Issue Number>_main_<Version>_<Description>`
- [ ] Fix the bug and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **main_\<Version>**.
Pull request: `bugfix <Issue Number> main_<Version> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, **Milestone**, and **Linked issues**
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Complete the steps above to fix the bug on the **develop** branch.
Branch name:  `bugfix_<Issue Number>_develop_<Description>`
Pull request: `bugfix <Issue Number> develop <Description>`
- [ ] Close this issue.
